README.md

```diff
@@ -1 +1,25 @@
-# knotx-adapter-archetype
\ No newline at end of file
+![Knot.x logo](https://avatars2.githubusercontent.com/u/24564424?v=3&s=400)
+
+# Knot.x Extension Maven Archetype
+
+Use this archetype to generate a custon _Knot.x_ extension.
```

Comment:
This is also very good start for the project that uses Knot.x . Action adapter can't be as easily extracted as Service adapter to generic Http implementation, so usually project needs to implement its version that handles POSTs.